requirements.txt

```diff
@@ -2,7 +2,7 @@ matplotlib==2.2.4
 Pillow==6.2.0
 scikit-learn==0.20.3
 scipy==1.2.1
-tensorflow==1.15.2
+tensorflow==2.3.0
```

Comment:
just merged in change to alter this dep to 2.4.0 so you can remove this line from this PR and i'll get it in.

Reply:
ok, just updated the PR, should be fine now